arch/arm/src/samv7/sam_serial_spi.c

```diff
@@ -69,16 +69,16 @@
 
 struct sam_serial_spi_pins_s
 {
-  uint32_t mosi;
-  uint32_t miso;
-  uint32_t sck;
-  uint32_t nss;
+  const uint32_t mosi;
+  const uint32_t miso;
+  const uint32_t sck;
+  const uint32_t nss;
 };
 
 struct sam_serial_spi_s
 {
   struct sam_serial_spi_pins_s pins;
-  uint32_t base;               /* SPI controller register base address */
+  const uint32_t base;         /* SPI controller register base address */
```

Comment:
ditto

Reply:
I can revert. It was just from logical perspective as the values of the base address should not change